src/components/atoms/link.tsx

```diff
@@ -1,27 +1,46 @@
 import React from "react";
+import { redirectTo } from "../../core/utils/helpers/url";
 
 export interface LinkProps {
   href: URL;
   children: React.ReactNode;
   isNewTab?: boolean;
   className?: string;
   id?: string;
+  trackClick?: () => void;
 }
 
 export const Link: React.FC<LinkProps> = ({
   href,
   children,
   isNewTab,
   className,
-  id
+  id,
+  trackClick
 }) => {
+  const redirect = () => {
+    redirectTo(href);
+  };
+
   return (
+    // If we want to track this click we have to first NOT set the href attribute
+    // & instead redirect through an onclick handler
+    // eslint-disable-next-line jsx-a11y/anchor-is-valid
     <a
       id={id}
-      href={String(href)}
+      // eslint-disable-next-line no-script-url
+      href={trackClick ? "javascript:void(0)" : String(href)}
```

Comment:
Please reconsider this.

[The `jsx-a11y` documentation for this linting rule](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/anchor-is-valid.md) suggests alternative solutions which improve accessibility.

Reply:
Is there a reason this is not a proper `button` element instead of `<span role="button">`?

It seems as if [this would be preferable to the custom onClick + onKeyUp handling](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role#sect1).

Reply:
Also remember to pass the `className` attribute to the button.

Reply:
Also remember to support the `target` attribute. This may require you to use [`Window.open()`](https://developer.mozilla.org/en-US/docs/Web/API/Window/open).

Reply:
Also remember that the tracking function should take the waiting into account so tracking will occur before the browser is redirected.

Reply:
@kasperg I tried to make it a button, and then when I did I saw the reason why I didn't make it a button in the first place. It's because it was never meant to be one (see picture) so the styling is all over the place.
![image](https://user-images.githubusercontent.com/28546954/203372564-ec9e5579-df69-4d09-a741-14440b46e4b8.png)
So if we do make it a button then we first have to go back to the design system and accommodate for that. It's not a problem at all, it will just consume extra time. And considering we want to present this to DDF on Thursday at noon, I'd say we rather utilize a custom role="button" span.
I'm open to your suggestion if you feel strongly about it, pls let me know back ;)